cms/envs/common.py

```diff
@@ -935,6 +935,24 @@
 
 ################ ADVANCED_COMPONENT_TYPES ###############
 
+# These strings are entry-point names from the setup.py of the XBlock.
+# For example:
+#
+#   setup(
+#       name='xblock-foobar',
+#       version='0.1',
+#       packages=[
+#           'foobar_xblock',
+#       ],
+#       entry_points={
+#           'xblock.v1': [
+#               'foobar-block = foobar_xblock:FoobarBlock',
```

Comment:
It's a little unclear that you're getting 'foobar-xblock' at first glance from the left side of this assingment. Maybe go crazy and make this `xyzzy` to be very clear where you are getting the name from.